### PR TITLE
cloudformation-guard 0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by johnttompkins**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
[Yes. Only ones are closed because of the previous beta tag](https://github.com/Homebrew/homebrew-core/pull/58364)
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
rebased to the master to clear the code style failure caused by another formula bottle commit.

**Review by chenrui333 — DISMISSED** [review]

File: Formula/cloudformation-guard.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  desc "Checks CloudFormation templates for compliance using a declarative syntax"
+  homepage "https://github.com/aws-cloudformation/cloudformation-guard"
+  url "https://github.com/aws-cloudformation/cloudformation-guard/archive/v0.7.0.tar.gz"
+  sha256 "f285c76ca32ca671380f254d0ce01dc6e5e3d91ef7d6e4c7fc2d6abeb7520e79"
+  license "Apache-2.0"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "./cfn-guard"
```
Comment (dawidd6) [inline Formula/cloudformation-guard.rb:11]:
```suggestion
    cd "cfn-guard" do
      system "cargo", "install", *std_cargo_args
    end
```

Comment (jotompki) [reply Formula/cloudformation-guard.rb:11]: ah great this is way better. thanks

Comment (chenrui333) [reply Formula/cloudformation-guard.rb:11]: 👍  (also please squash the commits into one :) )

Comment (jotompki) [reply Formula/cloudformation-guard.rb]: rebased and squashed

**Review by chenrui333 — APPROVED** [review]

**Comment by jotompki** [comment]
When can we get this merged into homebrew/core? Any outstanding requests?

**Comment by Rylan12** [comment]
Thanks, @jotompki!

This was merged in 1875a1cbdf20d83b555415f53fc4d1d3515a0a7d

**Comment by jotompki** [comment]
Any way to get some visibility into what could be causing the build errors that I am seeing here:

https://formulae.brew.sh/formula/cloudformation-guard

**Pull request closed without merging**